lib/Frontend/DiagnosticVerifier.cpp

```diff
@@ -128,7 +128,7 @@ findDiagnostic(std::vector<CapturedDiagnosticInfo> &CapturedDiagnostics,
 
     // Verify the classification and string.
     if (I->Classification != Expected.Classification ||
-        I->Message.find(Expected.MessageStr) == StringRef::npos)
+        !(I->Message.contains(Expected.MessageStr)))
```

Comment:
in fact in most of these places, I don't think you need the extra () for the !. The ! should bind tight I think.

Reply:
Thank you! I removed the extra () for ! everywhere :)